qiskit/pulse/schedule.py

```diff
@@ -1969,8 +1969,7 @@ def _collect_scoped_parameters(
 
         if filter_regex and not re.search(filter_regex, new_name):
             continue
-        scoped_param = Parameter.__new__(Parameter, new_name, uuid=getattr(param, "_uuid"))
-        scoped_param.__init__(new_name)
+        scoped_param = Parameter(new_name, uuid=getattr(param, "_uuid"))
```

Comment:
I didn't look into it - I'd assume that at some point it was possible that the `_uuid` field wasn't set on some objects, but it could just have been a mistake.